ui/admin/app/routes/scopes/scope/roles/role/index.js

```diff
@@ -4,5 +4,25 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class ScopesScopeRolesRoleIndexRoute extends Route {}
+export default class ScopesScopeRolesRoleIndexRoute extends Route {
+  // =services
+
+  @service store;
+
+  // =methods
+
+  /**
+   * Pre-loads all scopes since they will be necessary in sub-routes.
+   * @param {RoleModel} role
+   */
+  async afterModel(role) {
+    if (role.scope.isGlobal) {
```

Comment:
Just wanted to double check, how come we only grab scopes at the global scope? What do we do for org level roles?

Reply:
ah yes, great catch! i forgot that today i refactored the manage-org-projects page to also use indexed db so this need to be updated as well.